opentelemetry-sdk/src/opentelemetry/sdk/trace/sampling.py

```diff
@@ -212,6 +258,20 @@ def should_sample(
     def get_description(self) -> str:
         pass
 
+    # TODO(sconover) added close to all samplers, because of cleanup needed
+    # for RemoteControlledSampler
+    # Q: Where should sampler.close() be called? I believe it might be 
+    # TracerProvider#shutdown (but not entirely sure)
+    def close(self) -> None:
```

Comment:
@srikanthccv Existing samplers in this repo are simple and don't need an explicit lifecycle event like this. Are we ok with me proposing to add a close method like this...and then, to call that in something like `TracerProvider#shutdown`?